### PR TITLE
For #298: Added test checking that manager does not have access to

### PR DESCRIPTION
**Pull request by paulodamaso**
For #298: 
- Added `Role` to `Employee`
- Added test for checking if `Manager` can't access `Director`
- Left todo for continue the implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#321](https://github.com/timelesslounge/timelessis/issues/321) is now in scope, role is `REV`

**Comment by 0crat** [comment]
Job `gh:timelesslounge/timelessis#321` already assigned to @vladarefiev, can't assign to @vladarefiev

**Comment by 0crat** [comment]
This pull request [#321](https://github.com/timelesslounge/timelessis/issues/321) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/f3aa9452-5c7d-4d5a-a3d9-0beb310c2e4a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: tests/integration/it_manager_privilages_test.py [hunk 1]
```diff
@@ -99,11 +111,62 @@ def test_can_access_same_company_employees(app, db_session):
...
     db_session.add(other)
     db_session.commit()
     assert has_privilege(
         method=Method.READ, resource="employee", employee_id=other.id
     )
 
+@pytest.mark.skip
+def test_manager_cant_access_director(app, db_session):
+    """
+    @todo #298:30min Add role to the employee model and check that Manager
```
Comment (vladarefiev) [inline tests/integration/it_manager_privilages_test.py]: @paulodamaso `role to the employee model` role is already added as I can see

File: tests/integration/it_manager_privilages_test.py [hunk 2]
```diff
@@ -86,7 +98,7 @@ def test_can_access_same_company_employees(app, db_session):
         user_status="on",
         registration_date=datetime.utcnow(),
         company_id=my_company.id,
-        email="test@test.com", password="bla"
+        email="test@test.com", password="bla",role_id=manager_role.id
```
Comment (vladarefiev) [inline tests/integration/it_manager_privilages_test.py]: @paulodamaso pls add empty space or new line between `password="bla",role_id=manager_role.id`

File: tests/integration/it_manager_privilages_test.py [hunk 3]
```diff
@@ -99,11 +111,62 @@ def test_can_access_same_company_employees(app, db_session):
...
+    me = Employee(
+        id=1, first_name="Alice", last_name="Cooper",
+        username="alice", phone_number="1",
+        birth_date=datetime.utcnow(),
+        pin_code=7777,
+        account_status="on",
+        user_status="on",
+        registration_date=datetime.utcnow(),
+        company_id=my_company.id,
+        email="test@test.com", password="bla",role_id=manager_role.id
```
Comment (vladarefiev) [inline tests/integration/it_manager_privilages_test.py]: @paulodamaso the same

**Comment by vladarefiev** [comment]
@paulodamaso good to me. but only a few small comments and let's merge.

**Comment by paulodamaso** [comment]
@vladarefiev Thanks for the review, fixed the issues, please take a look

**Comment by vladarefiev** [comment]
@paulodamaso thanks

**Comment by vladarefiev** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 20 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#321](https://github.com/timelesslounge/timelessis/issues/321) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one major issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vladarefiev** [comment]
@ypshenychka Correct

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**